.circleci/config.yml

```diff
@@ -1,23 +1,69 @@
 version: 2
 workflows:
   version: 2
-  main:
+  tests:
     jobs:
-      - compile
+      - compile:
+          filters:
+            tags:
+              only: /.*/
+      - publish_image:
+          requires:
+            - compile
+          filters:
+            branches:
+              ignore: /.*/
```

Comment:
will this ignore master?

Reply:
nvm this is fine if we are going to start tagging this repo

Reply:
Right, exactly.